app/assets/src/components/views/SampleUploadFlow/host_organism_message.scss

```diff
@@ -15,12 +17,25 @@
 .messageLine {
   margin: 0 $space-s;
   padding: $space-s;
-
-  &:nth-child(1) {
-    margin-top: $space-s;
-  }
+  font-size: $font-size;
+  margin: 0 44px; // align to right of icon
```

Comment:
Can this be a space variable?

Reply:
No, because it depends on `notification.scss` which itself does not use variables. Would you recommend changing `notification.scss` to do so?

Reply:
Yes, if simple enough... in general, something we try to do is keep updating the existing components for new methods/rules. But it is up to you to evaluate the effort.